wagtail/contrib/modeladmin/tests/test_simple_modeladmin.py

```diff
@@ -695,3 +695,18 @@ def test_model_with_single_tabbed_panel_and_edit_handler(self):
         # clean up for future checks
         delattr(Publisher, 'content_panels')
         delattr(Publisher, 'edit_handler')
+
+
+class TestPrepopulatedFields(TestCase, WagtailTestUtils):
+
+    def setUp(self):
+        self.login()
+
+    def get(self, **params):
+        return self.client.get('/admin/modeladmintest/prepopulatedfieldmodel/create/', params)
+
+    def test_in_context(self):
+        response = self.get()
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn('data-prepopulated-fields="[{&quot;id&quot;: &quot;#id_slug&quot;, &quot;name&quot;: &quot;slug&quot;, &quot;dependency_ids&quot;: [&quot;#id_name&quot;], &quot;dependency_list&quot;: [&quot;name&quot;], &quot;maxLength&quot;: 50, &quot;allowUnicode&quot;: false}]"', response.content.decode('UTF-8'))
```

Comment:
Instead of writing a whole new class for this feature, I feel we can get away with just adding a single test to the existing `TestCreateView` is this module (if we also go ahead and reuse the `Book` model here. Does that sound okay to you @dbramwell ?